pkg/estimator/pb/types.go

```diff
@@ -47,3 +49,45 @@ type MaxAvailableReplicasResponse struct {
 	// +required
 	MaxReplicas int32 `json:"maxReplicas" protobuf:"varint,1,opt,name=maxReplicas"`
 }
+
+// MaxUnschedulableReplicasRequest represents the request that sent by gRPC client to calculate unschedulable replicas.
+type MaxUnschedulableReplicasRequest struct {
+	// Cluster represents the cluster name.
+	// +required
+	Cluster string `json:"cluster" protobuf:"bytes,1,opt,name=cluster"`
+	// Resource represents the Kubernetes resource to be propagated.
+	// +required
+	Resource ObjectReference `json:"resource" protobuf:"bytes,2,opt,name=resource"`
+	// Resource represents the Kubernetes resource to be propagated.
+	// +optional
+	UnscheduableThreshold time.Duration `json:"unscheduableThreshold,omitempty" protobuf:"varint,3,opt,name=unscheduableThreshold,casttype=time.Duration"`
```

Comment:
```suggestion
	UnschedulableThreshold time.Duration `json:"unschedulableThreshold,omitempty" protobuf:"varint,3,opt,name=unschedulableThreshold,casttype=time.Duration"`
```
s/Unscheduable/Unschedulable/

Reply:
The comments of `UnschedulableThreshold` are not correct. (Don't know why lint ignored this. :( )

Reply:
Sorry, I alse made this mistake in proposal.